doc/plutus/explanations/plutus-tools-component-descriptions.rst

```diff
@@ -0,0 +1,321 @@
+.. _plutus_tools_component_descriptions:
+
+Plutus tools component descriptions
+=================================================
+
+The Plutus team has provided a set of “Tools”---components, libraries and 
+executables---for external developers to use. They are located in the 
+`plutus-apps <https://github.com/input-output-hk/plutus-apps>`_ repository. 
+
+For each tool listed below, we have indicated its specific location within 
+plutus-apps, provided a short description, indicated its dependencies, and given 
```

Comment:
I don't think it's that useful to list the dependencies. You'll have to maintain them otherwise they can easily get out of date.

Reply:
Yes, this seems kind of crazy to me! People who care can look in the cabal files! We _really_ don't want to duplicate information like that.

Reply:
Great point. I'll discuss with Konstantinos.

Reply:
After discussing with Konstantinos, we have removed "Dependencies" throughout this page since that info is too detailed for this page, and as Ziyang and Michael pointed out, too difficult to maintain.